scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/UploadServlet.java

```diff
@@ -108,11 +128,11 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
             while (i.hasNext()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)